x11/mesa/Portfile

```diff
@@ -36,48 +36,49 @@ meson.wrap_mode         nodownload
 set py_ver              3.12
 set py_ver_nodot        [string map {. {}} ${py_ver}]
 
-depends_build-append    port:pkgconfig \
+depends_build-append    path:bin/pkg-config:pkgconfig \
                         port:flex \
                         port:bison \
-                        port:py${py_ver_nodot}-mako \
                         port:python${py_ver_nodot} \
+                        port:py${py_ver_nodot}-mako \
+                        port:py${py_ver_nodot}-yaml \
                         port:xorg-xorgproto
 
 depends_lib-append      port:expat \
                         port:xorg-libxcb \
                         port:xorg-libX11 \
                         port:xorg-libXext \
                         port:xorg-libXdamage \
+                        port:xorg-libXrandr \
                         port:zlib \
                         port:zstd
 
 patchfiles              patch-meson-spec-python.diff \
-                        patch-driver_extensions.diff \
-                        patch-pre-10.8-scandir.diff
+                        patch-pre-10.8-scandir.diff \
+                        patch-fix-build.diff \
+                        patch-fix-linking.diff
```

Comment:
Ah, it is our patch which was breaking the build. It should go inside LLVM variant.